server/resp/src/main/java/org/infinispan/server/resp/RespCommand.java

```diff
@@ -89,7 +90,7 @@ public String getName() {
       indexedRespCommand[12] = new RespCommand[]{new MGET(), new MSET()};
       indexedRespCommand[15] = new RespCommand[]{new PUBLISH(), new PING(), new PSUBSCRIBE(), new PUNSUBSCRIBE()};
       indexedRespCommand[16] = new RespCommand[]{new QUIT()};
-      indexedRespCommand[17] = new RespCommand[]{new RESET(), new READWRITE(), new READONLY()};
+      indexedRespCommand[17] = new RespCommand[]{new RESET(), new READWRITE(), new READONLY(), new RPUSH(), new RPUSHX()};
```

Comment:
These new commands can go before the others.
```suggestion
      indexedRespCommand[17] = new RespCommand[]{new RPUSH(), new RPUSHX(), new RESET(), new READWRITE(), new READONLY()};
```